src/functionalTest/java/uk/gov/hmcts/reform/refunds/functional/RefundsApproverJourneyFunctionalTest.java

```diff
@@ -2018,8 +2018,8 @@ public void positive_refund_reject_reason_unable_to_apply_refund_to_card_and_ema
         assertThat(updateRefundStatusResponse.getStatusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
 
         // verify that contact details is erased
-        RefundDto refundDto = getRejectRefundDto(refundReference, "Rejected");
-        assertEquals(RefundStatus.REJECTED, refundDto.getRefundStatus());
+        RefundDto refundDto = getRejectRefundDto(refundReference, "Approved");
+        assertEquals(RefundStatus.APPROVED, refundDto.getRefundStatus());
```

Comment:
This functional test is for reject refund and we are passing for reject refund for update status but this asset is approved and I feel this is wrong. Also as we per test case name and scenario checking approved is wrong. Could you please correct and if required please add new functional test case for approved refund.